java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5663_java.html

```diff
@@ -1,39 +1,17 @@
-<div class="paragraph">
```

Comment:
I have some doubts about this one. I feel that's the one generated from Github. What was the conclusion of the discussion in LT? Should we keep all these useless classes?

I also see these useless nodes when using `gh_generate` command, but I kind of remember that we said they should be gone. Am I using an outdated version of rule-api?

Reply:
I think this question wasn't resolved yet, whether we need to keep additional decorations from asciidoctor or not.
As for this rule. I tested it locally and on peach and it looks pretty much the same as others.

On the other hand, to feel more confident we can keep the version from old-fasioned Jira, because we still don't know whether it will break anything else or not.

So I'd rather use the version generated directly from Jira at the moment to stay safe.